### PR TITLE
[enterprise-4.12] OBSDOCS-69: Reorganize steps in Query Browser procedure

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `enterprise-4.12`

Issue: [OBSDOCS-69](https://issues.redhat.com/browse/OBSDOCS-69)

Links to docs preview:

- [Querying metrics for all projects as a cluster administrator](https://72500--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/querying-metrics#querying-metrics-for-all-projects-as-an-administrator_querying-metrics)
- [Exploring the visualized metrics](https://72500--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/querying-metrics#exploring-the-visualized-metrics_querying-metrics)

QE review:
- [x] QE has approved this change.

Peer review:
- [x] Peer review done

Additional information: 
**This issue is created against `enterprise-4.12` branch because newer versions already have this change implemented.**

### CODE REVIEW

**Comment by dfitzmau** [comment]
/remove-label peer-review-in-progress

/label peer-review-done

LGTM but added some minor comments inline.

File: modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc [hunk 1]
```diff
@@ -16,35 +16,35 @@ As a cluster administrator or as a user with view permissions for all projects,
...
-You can also move your mouse pointer over a suggested item to view a brief description of that item.
-====
+|===
+|Option |Description
+
+|Create a custom query.
+|Add your Prometheus Query Language (PromQL) query to the *Expression* field.
+
+As you type a PromQL expression, autocomplete suggestions are displayed in a list. These suggestions include functions, metrics, labels, and time tokens.
+You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression. You can also move your mouse pointer over a suggested item to view a brief description of that item.
```
[the remark this answers is not in the document]
Comment (eromanova97) [reply modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc:30]: As per IBM: https://www.ibm.com/docs/en/ibm-style?topic=elements-keyboard-keys#key-names we refer to key names without using bold. Thank you!

File: modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc [hunk 2]
```diff
@@ -16,35 +16,35 @@ As a cluster administrator or as a user with view permissions for all projects,
...
 
-. To run queries that you created, select *Run Queries*.
+. To run queries that you created, click *Run queries*.
 The metrics from the queries are visualized on the plot.
 If a query is invalid, the UI shows an error message.
 +
 [NOTE]
 ====
-Queries that operate on large amounts of data might time out or overload the browser when drawing time series graphs. To avoid this, select *Hide graph* and calibrate your query using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
+Queries that operate on large amounts of data might time out or overload the browser when drawing time series graphs. To avoid this, click *Hide graph* and calibrate your query using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
```
[the remark this answers is not in the document]
Comment (eromanova97) [reply modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc]: Done, thank you!

**Comment by mburke5678** [comment]
@eromanova97 My only concern is that the _[Querying metrics](https://docs.openshift.com/container-platform/4.13/monitoring/managing-metrics.html#querying-metrics-for-all-projects-as-an-administrator_managing-metrics)_ module in 4.13+ use `Options menu {kebab}` with a capital O, rather than the lower-case o that you have here. Let me know if you want to fix it here and we can merge.

**Comment by eromanova97** [comment]
Hello @mburke5678 I understand your concern here. I had a similar concern, the IBM recommendation is this:
"If the graphical item does not have a label or name, provide a generic description. Ex: Click the ellipsis button [icon of the button]." [link](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#references-to-interface-elements)

That is why I included it with lowercase. So I think the question is now if we want to use the recommended naming or if we want to keep it uppercase for consistency. Another option can be that I can change the uppercase to lowercase in newer versions. WDYT?

**Comment by mburke5678** [comment]
@eromanova97 It seems that captial O is more common in our docs. Could you make the change here. then maybe take the issue to the broader docs team?

**Comment by eromanova97** [comment]
I agree that if it is more common in our docs, then the capital O should be fine as the mention in IBM is more of a recommendation in this case. I made the changes, thank you!

**Pull request merged**